src/bp/ui-studio/src/web/reducers/flows.ts

```diff
@@ -239,7 +239,7 @@ const doCreateNewFlow = name => {
     }
   ]
 
-  const isSubWorkflow = window.USE_ONEFLOW && parseFlowName(name).folders?.length
+  const isSubWorkflow = window.USE_ONEFLOW && parseFlowName(name).parentWorkflowPath
```

Comment:
Didn't you remove all references of parent?

Reply:
This got changed in the ui implementation